index.html

```diff
@@ -0,0 +1,88 @@
+                                          aria-describedby="help-text-counter"
+                                          placeholder="Enter text here..."></textarea>
+                                <div id="help-text-counter" class="form-text">The text must consist of at least one
+                                    word.
+                                </div>
+                            </div>
+                            <div class="d-flex justify-content-between">
+                                <button type="button" class="btn btn-dark calculate-btn">Calculate quantity</button>
+                                <button type="button" class="btn btn-secondary random-bg-btn">Random Background</button>
+                            </div>
+                        </form>
+                    </div>
+                    <div class="col-12">
+                        <div class="alert alert-danger mt-3 text-error" role="alert">
+                            Text field cannot be empty!
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="row justify-content-end">
+                    <div class="col-10">
+                        <div class="card text-white bg-dark mb-3 card-container">
+                            <div class="card-header">Sentences</div>
+                            <div class="card-body">
+                                <h5 class="card-title">Amount of sentences</h5>
+                                <p class="card-text">quantity: <span class="sentences-count">0</span></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-10">
+                        <div class="card text-white bg-secondary mb-3 card-container">
+                            <div class="card-header">Words</div>
+                            <div class="card-body">
+                                <h5 class="card-title">Amount of words</h5>
+                                <p class="card-text">quantity: <span class="words-count">0</span></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-10">
+                        <div class="card bg-light mb-3 card-container">
+                            <div class="card-header">Letters</div>
+                            <div class="card-body">
+                                <h5 class="card-title">Amount of letters</h5>
+                                <p class="card-text">quantity: <span class="letters-count">0</span></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</main>
+<footer class="py-4">
+    <div class="mx-4 px-5">
+        <p>© TUMO Labs - GSL 2024 / All Rights Reserved</p>
+    </div>
+</footer>
+</body>
+</html>
```

Comment:
Html Kod@ grvac er hiasqanc uxxaki kareli er grel mi poqr aveli parz, bayc @ndhanur shat lav e.

Reply:
I wrote the code using the bootstrap library, and this is the structure of this library.
I agree with you, if I didn't use the library, I could write simpler HTML, but in this case I would have to write all the styles myself.